docs/features/search/README.md

```diff
@@ -24,22 +24,20 @@ Backstage ecosystem.
 
 ## Project roadmap
 
-| Version                           | Description                                                                                                                                                        |
-| --------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Backstage Search v0 ✅            | Search Frontend letting you search through the entities of the software catalog. [See v0 Use Cases.](#backstage-search-v0)                                         |
-| [Backstage Search V0.5 ✅ ][v0.5] | Foundations for the architecture.                                                                                                                                  |
-| [Backstage Search v1 ⌛][v1]      | Basic “out-of-the-box” in-memory indexing process of entities, and their metadata, registered to the Software Catalog. [See v1 Use Cases.](#backstage-search-v1)   |
-| [Backstage Search v2 ⌛][v2]      | Search Backend responsible for the indexing process of entities, and their metadata, registered to the Software Catalog. [See v2 Use Cases.](#backstage-search-v2) |
-| [Backstage Search v3 ⌛][v3]      | Standardized Search API lets you index other plugins data to the search engine of choice. [See v3 Use Cases.](#backstage-search-v3)                                |
-
-[v0.5]: https://github.com/backstage/backstage/milestone/25
-[v1]: https://github.com/backstage/backstage/milestone/26
-[v2]: https://github.com/backstage/backstage/milestone/27
-[v3]: https://github.com/backstage/backstage/milestone/28
+| Version                            | Description                                                                                                                                                            |
+| ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Backstage Search Pre-Alpha ✅      | Search Frontend letting you search through the entities of the software catalog. [See Pre-Alpha Use Cases.](#backstage-search-pre-alpha)                               |
+| [Backstage Search Alpha ⌛][alpha] | Basic “out-of-the-box” in-memory indexing process of entities, and their metadata, registered to the Software Catalog. [See Alpha Use Cases](#backstage-search-alpha). |
```

Comment:
The changeset seems to suggest the "basic out of the box in memory indexing" is already provided via pre-alpha, is that right? if so maybe some wording tweaks to the Alpha stage.

Reply:
Oh, yeah.  Thanks for noting!  Took a look at this doc with fresh eyes and quite a bit needs to be shuffled around...